contracts/core/modules/follow/ApprovalFollowModule.sol

```diff
@@ -118,15 +127,21 @@ contract ApprovalFollowModule is IFollowModule, FollowValidatorFollowModuleBase
      * @param profileOwner The profile owner of the profile to query the approvals with.
      * @param profileId The token ID of the profile to query approvals with.
      * @param toCheck The address array to query approvals for.
+     *
+     * @return bool[] Whether the addresses are approved or not.
```

Comment:
Similar to above.

Reply:
Resolved in commit 2c30269f1a7b1fac30cae6934ed23d692e54ebb6!